spec/support/matchers.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
likewise, I wonder if I should work on matchers like this for tests (like we have in cocina_models) to be provided by SdrViewComponents